src/misc.c

```diff
@@ -92,12 +92,19 @@ const char * get_ip_source(const struct _u_request * request) {
 char * get_client_hostname(const struct _u_request * request) {
   const char * ip_source = get_ip_source(request);
   char * hostname = NULL;
-  struct hostent * lh;
   
   if (ip_source != NULL) {
-    lh = gethostbyname(ip_source);
-    if (lh) {
-      hostname = msprintf("%s - %s", ip_source, lh->h_name);
+    struct addrinfo hints = { .ai_family = AF_UNSPEC, .ai_flags = AI_CANONNAME };
```

Comment:
Even though it's no longer mandatory, I'm used to declare all variables at the beginning of the functions, I find it easier to read and to miss multiple variables with the same name.